Sources/Amplitude/Amplitude.m

```diff
@@ -952,10 +910,10 @@ - (void)makeEventUploadPostRequest:(NSString *)url events:(NSString *)events num
                     // success, remove existing events from dictionary
                     uploadSuccessful = YES;
                     if (maxEventId >= 0) {
-                        (void) [self.dbHelper removeEvents:maxEventId];
+                        [AMPStorage remove:[AMPStorage getDefaultEventsFile]];
```

Comment:
Do we need to handle the _eventsBuffer in this case?

Reply:
Good catch.